package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tcomb-builder",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Would expect this to be at a minor version bump because we're adding new backwards compatible APIs.